### PR TITLE
Story #214

### PR DESCRIPTION
**Pull request by alecpm**

### CODE REVIEW

File: src/vms/static/app/hours.js [hunk 1]
```diff
@@ -53,6 +64,17 @@ var VolunteerHoursForm = React.createClass({
 					</td>
 				)}
 			</tr>
+			<tr style={{display: reg.time_exception ? 'table-row' : 'none'}}>
+				<td colSpan={1 + days.length} style={{padding: 0}}>
+					<div style={{padding: "14.66667px 11px"}}>
```
Comment (cewing) [inline src/vms/static/app/hours.js]: Is it okay to put style declarations inline in the HTML like this?

Comment (alecpm) [inline src/vms/static/app/hours.js]: Yeah, it seems to be pretty standard practice in the React world (though they like to put them into separate objects which can be reused), and not an anti-pattern like everywhere else in HTML world.  It's also coming straight from the design mockups as an inline style.

**Comment by cewing** [comment]
Works for me, then.  Merging

File: src/vms/static/app/hours.js (hunk 1 above)
Comment (davisagli) [inline src/vms/static/app/hours.js]: Yeah, it's less horrendous than most inline styles, because having it in a React component avoids repetition.

**Pull request merged**